### PR TITLE
Videobuffer

### PR DESCRIPTION
**Pull request by FernetMenta**

### CODE REVIEW

File: xbmc/cores/VideoPlayer/Process/VideoBuffer.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#define MAX_PLANES 3
+
+typedef struct YuvImage
+{
+  uint8_t* plane[MAX_PLANES];
```
Comment (notspiff) [inline xbmc/cores/VideoPlayer/Process/VideoBuffer.h]: suggestion: static const int MAX_PLANES = 3 as a member. context is always good.

Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/Process/VideoBuffer.h]: good idea, will do

**Comment by hudokkow** [comment]
NP.
Please see if https://github.com/hudokkow/xbmc/commit/476fea418fa1b1ed37e34e7572c9be55d201ff81 is something that interests you.

**Comment by FernetMenta** [comment]
@hudokkow yes, please submit to mainline. I already removed a lot of those in the past but obviously not all.

**Comment by hudokkow** [comment]
Will do, thx.

**Pull request merged**